docs/user_manual.md

```diff
@@ -401,6 +402,13 @@ Will ultimately have these options at `rolling-update` time.
 Since timeout, migrate, and token weren't specified either via CLI or job config, they get their
 values from the defaults in [`RolloutOptions`][rollout-options-code].
 
+#### runtime
+Optional. Container runtime to use. When specifying a runtime, you will need to ensure that 
+the docker daemon knows about the specified runtime. For example, to enable nvidia runtime,
+you will need to configure the host to install runtime.
```

Comment:
@vbhavsar, is "runtime" word at the end of this sentence correct? Should it be "nvidia" or "nvidia driver" instead?

Reply:
It is a bit confusing to read. I will change the wording.